erpnext/hr/doctype/salary_structure/salary_structure.py

```diff
@@ -29,12 +27,6 @@ def get_ss_values(self,employee):
 	def validate_amount(self):
 		if flt(self.net_pay) < 0 and self.salary_slip_based_on_timesheet:
 			frappe.throw(_("Net pay cannot be negative"))
-
-	def validate_joining_date(self):
```

Comment:
keep the function and update it based on from and to date on salary structure employee table